src/AppKit/NSBezierPath.cs

```diff
@@ -110,5 +110,15 @@ public unsafe void AppendPathWithGlyphs (uint[] glyphs, NSFont font)
 				_AppendPathWithGlyphs ((IntPtr)ptr, glyphs.Length, font);
 		}
 
+		public unsafe void AppendBezierPathWithCGGlyphs (uint[] glyphs, NSFont font)
```

Comment:
Things were double confusing because we were not distinguishing between NSGlyph and CGGlyph correctly in "ancient" bindings.